day_two_guide/topics/api_service_status.adoc

```diff
@@ -24,6 +24,7 @@ oc get pod -n kube-system -o wide
 NAME                                               READY     STATUS    RESTARTS   AGE       IP            NODE
 master-api-myserver.com                            1/1       Running   0          7h        10.240.0.16   myserver.com/healthz
 
-$ curl -k https://myserver.com/healthz
+$ curl -k https://myserver.com:443/healthz <1>
 ok
 ----
+<1> The web console port in this example is `443`. This may differ, based on what is specified in the `openshift_master_console_port` variable in the host inventory file prior to OpenShift deployment. If the variable is not included, port `8443` is set by default.
```

Comment:
I'd s/This may differ, based on what/Specify the value that is

s/OpenShift/{product-title}  (You can't use "OpenShift" as a standalone term. The only time you can use "OpenShift" is if it's part of a specific component name. If you're referring to the product, you must use "{product-title}" to makes sure that the right terminology is always used in all the different distributions.)

Reply:
@kalexand-rh thank you. I have now pushed those changes.